compysition/actors/eventattributemodifier.py

```diff
@@ -177,4 +177,141 @@ def get_modify_value(self, event):
         return value
 
 class HTTPJSONAttributeModifier(JSONEventAttributeModifier, HTTPStatusModifier):
-    pass
\ No newline at end of file
+    pass
+
+class XMLEventAttributeModifier(EventAttributeModifier):
+
+    def get_key_chain_value(self, event, value):
+        keys = self.key.split(self.separator)
+        event_key = keys.pop(0)
+        if len(keys) == 0:
+            event.set(event_key, value)
+        else:
+            root_element = event.get(event_key, None)
+            current_element = root_element
+
+            try:
+                item = keys.pop(0)
+                if not item == current_element.tag:
+                    raise
+                while True:
+                    if len(keys) > 0:
+                        item = keys.pop(0)
+                        next_step = current_element.get(item, None)
+                        if not next_step:
+                            next_step = etree.Element(item)
+                            current_element.append(next_step)
+                        current_element = next_step
+                    else:
+                        current_element.text = str(value)
+                        break
+            except Exception as err:
```

Comment:
You generally want to consume only the exception to which your log message applies and let the rest bubble up or get caught.